lib/amazonka/src/Amazonka/Auth.hs

```diff
@@ -109,6 +111,7 @@ discover ::
 discover =
   runCredentialChain
     [ fromKeysEnv,
+      fromSSOCache,
```

Comment:
I don't think this will be necessary because we should be doing all the SSO lookup when we handle config files in `fromFileEnv`. The CLI doesn't document that it can do SSO from environment variables, so we can assume that it won't and that we won't need a `fromSSOEnv` either.